server/models/todo.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Todo extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  };
+  Todo.init({
+    title: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true
+      }
+    },
+    description: {
+      type: DataTypes.STRING
+    },
+    status: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true
+      }
+    },
+    due_date: {
+      type: DataTypes.DATE,
+      validate: {
+        isToday(value) {
```

Comment:
bisa pakai punya sequelize juga yah  
```suggestion
isAfter : {
    args: new Date().toString(),
    msg: 'Date must be greater than today'
}
```